src/main.rs

```diff
@@ -588,6 +618,13 @@ fn main() -> std::result::Result<(), Box<dyn std::error::Error + 'static>> {
             print_table(features, 8, 3);
             Ok(())
         }
-        _ => Err(anyhow!("npm install exited with non-zero status").into()),
+        _ => Err(anyhow!("npm install exited with non-zero status; run by hand to diagnose.")),
```

Comment:
This reminds me: in clawbang I ended up _always_ capturing build output. I only display the output if the user's verbosity settings are high enough _or_ if there was a build error – it strikes me that doing something similar here would be a nice improvement for another PR.